BoxContentSDK/BoxContentSDK/Categories/NSDate+BOXAdditions.m

```diff
@@ -49,8 +49,10 @@ - (NSString *)box_ISO8601String
         dateFormatter.includeTime = YES;
         dateFormatter.defaultTimeZone = [[NSTimeZone alloc] initWithName:@"UTC"];
     });
-    
-    return [dateFormatter stringFromDate:self];
+
+    @synchronized (dateFormatter) {
+        return [dateFormatter stringFromDate:self];
```

Comment:
lets set up the string here, inside the synchronized block, and then return it below.

That makes it easier to debug, for breakpoints etc